docs/resources/Webhook.md

```diff
@@ -97,10 +97,14 @@ Used to represent a webhook.
 }
 ```
 
-## Create Webhook % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/webhooks
+## Create Webhook 
 
 Creates a new webhook and returns a [webhook](#DOCS_RESOURCES_WEBHOOK/webhook-object) object on success. Requires the `MANAGE_WEBHOOKS` permission.
 
+```
+% POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/webhooks
+```
```

Comment:
These endpoints are intentionally formatted like this for the HTML rendering hosted on https://discord.com/developers/docs/resources/webhook#create-webhook

You should revert all of these changes.